normalization/test_http_multipart.py

```diff
@@ -0,0 +1,180 @@
+from framework import tester
+import itertools
+import textwrap
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2019 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+
+class ContentTypeTestBase(tester.TempestaTest):
```

Comment:
The tests are relatively simple, but there is still need to add a high-level test description. Just to show what was covered without requirement to inspect the code.

Reply:
Added an overall description of what the test suite is about at the top. And also short descriptions about what each test checks.